### PR TITLE
Adding the configurationPolicyAnnotations parameter

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/24114

### CODE REVIEW

**Review by mprahl — APPROVED** [review]

File: governance/policy_generator.adoc [hunk 1]
```diff
@@ -578,6 +578,10 @@ Any default value listed here is overridden for an entry in the policies array e
 | Optional.
 Array of categories to be used in the `policy.open-cluster-management.io/categories` annotation. The default value is `CM Configuration Management`.
 
+| policyDefaults.configurationPolicyAnnotations
+| Optional.
+Key-value pairs of annotations to set on generated configuration policies. For example, you can disable policy templates by defining the following parameter: `{"policy.open-cluster-management.io/disable-templates": "true"}`. The default value is empty (`{}`).
```
Comment (swopebe) [inline governance/policy_generator.adoc:593]: no need for period on line 582

Comment (dockerymick) [reply governance/policy_generator.adoc:593]: Thanks for reviewing Brandi. I think that is a fair change, I will have to remove the period from the Optional or Required lines. I'll create a new break line between parameter description and requirement status. Thoughts?

Comment (swopebe) [reply governance/policy_generator.adoc:593]: I handle two incomplete sentences in one cell the same as comma separated values, however, on this you may consider another column for Requirements.

If not, I do a comma in between the two. I like the way your strategy looks and if it works, we can make this change across tables. (I don't like too many bullets in a list if we can avoid it, this looks good!)

**Review by swopebe — APPROVED** [review]

File: governance/policy_generator.adoc (hunk 1 above)
Comment (dockerymick) [reply governance/policy_generator.adoc:593]: Thanks Brandi! Likewise, I am not a fun of too many bullets. I would like to make the change across the doc. I'll bring this up in the channel or next meeting

**Comment by dockerymick** [comment]
Merging after receiving approval

**Pull request merged**